packages/frint-config/src/externals.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Webpack's `externals` equivalent object,
+ * listing dependencies that Frint packages use.
+ */
+export default {
+  // lodash
+  'lodash/cloneDeep': { root: ['_', 'cloneDeep'] },
+  'lodash/each': { root: ['_', 'each'] },
+  'lodash/find': { root: ['_', 'find'] },
+  'lodash/findIndex': { root: ['_', 'findIndex'] },
+  'lodash/first': { root: ['_', 'first'] },
+  'lodash/includes': { root: ['_', 'includes'] },
+  'lodash/isArray': { root: ['_', 'isArray'] },
+  'lodash/isEqual': { root: ['_', 'isEqual'] },
+  'lodash/isPlainObject': { root: ['_', 'isPlainObject'] },
+  'lodash/last': { root: ['_', 'last'] },
+  'lodash/get': { root: ['_', 'get'] },
+  'lodash/set': { root: ['_', 'set'] },
+  'lodash/mapValues': { root: ['_', 'mapValues'] },
+  'lodash/merge': { root: ['_', 'merge'] },
+  'lodash/omit': { root: ['_', 'omit'] },
+  'lodash/padStart': { root: ['_', 'padStart'] },
+  'lodash/tail': { root: ['_', 'tail'] },
+  'lodash/take': { root: ['_', 'take'] },
+  'lodash/takeRight': { root: ['_', 'takeRight'] },
+  'lodash/zipObject': { root: ['_', 'zipObject'] },
+  'lodash/zipWith': { root: ['_', 'zipWith'] },
+
+  // rxjs
+  'rxjs/BehaviorSubject': { root: ['Rx', 'BehaviorSubject'] },
+  'rxjs/Observable': { root: ['Rx', 'Observable'] },
```

Comment:
Since we're importing `Observable` anyway, does it make sense to import individual operators? It won't reduce the bundle size but adds complexity to source files (assuming `Observable` includes all operators)

Reply:
I have answered this with an example code snippet here: https://github.com/Travix-International/frint/pull/316#discussion_r141275482